src/reformat.c

```diff
@@ -447,23 +447,23 @@ static avifResult avifImageYUVAnyToRGBAnySlow(const avifImage * image, avifRGBIm
                     }
 
                     if (image->depth == 8) {
-                        unormU[0][0] = uPlane[(uvJ * uRowBytes) + (uvI * yuvChannelBytes)];
-                        unormV[0][0] = vPlane[(uvJ * vRowBytes) + (uvI * yuvChannelBytes)];
-                        unormU[1][0] = uPlane[(uvJ * uRowBytes) + (uvI * yuvChannelBytes) + uAdjCol];
-                        unormV[1][0] = vPlane[(uvJ * vRowBytes) + (uvI * yuvChannelBytes) + vAdjCol];
-                        unormU[0][1] = uPlane[(uvJ * uRowBytes) + (uvI * yuvChannelBytes) + uAdjRow];
-                        unormV[0][1] = vPlane[(uvJ * vRowBytes) + (uvI * yuvChannelBytes) + vAdjRow];
-                        unormU[1][1] = uPlane[(uvJ * uRowBytes) + (uvI * yuvChannelBytes) + uAdjCol + uAdjRow];
-                        unormV[1][1] = vPlane[(uvJ * vRowBytes) + (uvI * yuvChannelBytes) + uAdjCol + vAdjRow];
+                        unormU[0][0] = ptrU8[(uvI * yuvChannelBytes)];
+                        unormV[0][0] = ptrV8[(uvI * yuvChannelBytes)];
+                        unormU[1][0] = ptrU8[(uvI * yuvChannelBytes) + uAdjCol];
+                        unormV[1][0] = ptrV8[(uvI * yuvChannelBytes) + vAdjCol];
+                        unormU[0][1] = ptrU8[(uvI * yuvChannelBytes) + uAdjRow];
+                        unormV[0][1] = ptrV8[(uvI * yuvChannelBytes) + vAdjRow];
+                        unormU[1][1] = ptrU8[(uvI * yuvChannelBytes) + uAdjCol + uAdjRow];
+                        unormV[1][1] = ptrV8[(uvI * yuvChannelBytes) + vAdjCol + vAdjRow];
```

Comment:
I forgot to note in the commit message that I also changed uAdjCol to vAdjCol in this line and in line 466. Since we always set uAdjCol and vAdjCol to the same value, this doesn't really matter in practice.